testsuite/python/h5md.py

```diff
@@ -30,7 +30,7 @@
 import espressomd.io.writer
 try:
     import h5py  # h5py has to be imported *after* espressomd (MPI)
-    skipIfMissingPythonPackage = ut.case._id
+    skipIfMissingPythonPackage = utx._id
```

Comment:
okay, i still don't get why this should be a private function

Reply:
I've renamed it to `no_skip` in 08a674fae